internal/admin/dashboard/static/css/dashboard.css

```
@@ -1891,15 +1891,74 @@ td.col-price {
+}
+
+.chart-view-btn.active {
+    background: var(--accent);
+    color: #fff;
+}
+
+.chart-view-btn svg {
+    width: 16px;
+    height: 16px;
+    fill: none;
+    stroke: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)